projects/hathor-wallet-headless/0001-multisig-create-mint-melt-token.md

```diff
@@ -0,0 +1,327 @@
+        melt: { available: number, locked: number ),
+      },
+    },
+  },
+}
+```
+
+Lets represent the response document as ` $ `, as each element of a list ` [*] `, we have:
+* `$.complete` -- that represents the completeness of signatures required to use the transaction
+* `$.tx.inputs[*].signed` -- that indicates this input has a signature
+* `$.tx.inputs[*].mine` -- that indicates this input belongs to this wallet
+* `$.tx.outputs[*].mine` -- that indicates this output belongs to this wallet
+* `$.balance.tokens` -- that contains the balance of each token in the transaction
+* `$.balance.authorities` -- that contains the balance of mint and melt for each token
+
+## Initialize MultiSig wallet without seed
+
+This wallet is used by the coordinator service to sign or sign and push the transaction given the signatures collected by the service.
+
+By relaxing the checks in `/wallet/start` endpoint logic a wallet can have in its configuration the `multisig` key without the `seeds` key. This configuration impairs the wallet to use `/wallet/p2sh/tx-proposal/get-my-signatures` endpoint, however it still allows the use of `/wallet/p2sh/tx-proposal/sign` and `/wallet/p2sh/tx-proposal/sign-and-push` endpoints because the signatures are provided in the call.
+
+This wallet can also be called the "seedless wallet".
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+Despite the current design provides a minimal set of operations to enable the EVM compatible bridge, it still lacks some operations like "create authority", "destroy authority", and "delegate authority", that could equip an organization to take full control over its assets in a MultiSig wallet. Also, the token creation don't supports NFT token.
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+- The addition of specialized commands is better than change the behavior of an existing command because this approach avoids produce breaking changes.
+
+# Prior art
+[prior-art]: #prior-art
+
+* This design is a complement for the  [EVM compatible bridge design](https://github.com/HathorNetwork/rfcs/blob/doc/evm-compatible-brigde/projects/evm-compatible-bridge/design.md#federation-service). 
+* The commands to create token, mint and melt includes a fine tuned authority creation after the PRs [#291](https://github.com/HathorNetwork/hathor-wallet-headless/pull/291) and [#293](https://github.com/HathorNetwork/hathor-wallet-headless/pull/293) in the wallet-headless.
+
+# Unresolved questions
+[unresolved-questions]: #unresolved-questions
+
+* Should NFT data be allowed in the create token endpoint for the MultiSig wallet, or do it deserves its own endpoint, as it happens for the single wallet?
+
+# Future possibilities
+[future-possibilities]: #future-possibilities
+
+* Create authority for a given token
+* Destroy authority for a given token
+* Delegate authority for a given token
+* Create NFT token
+
+## Create authority for a given token
```

Comment:
I didn't understand this feature, can you elaborate more?

Reply:
Some exchange or other partner may create authority to mint or melt and spread it though its wallets for the purpose to increase throughput or spread governance power.

Reply:
The feature described is a delegate authority, which is described in the other section.

Reply:
Oh, no. In the delegate authority we "transfer" the ownership of an authority without "create" new ones, right? In the create authority we are exchanging 1 authority to N authorities. They are not the same thing, except if it is possible to "create" N authorities with the delegate as well. If it is the case, you right.

Reply:
In the end, we always create new authorities because we must spend one authority utxo. By default we create 2 authorities, one new and one for yourself, but we can create 1 or 2 with the current lib code. I understand that the proposal is to be able to create more than 2, and I like the idea, I just think it's the same as the delegate authority, only more powerful.

Reply:
Ok. Vou descrever então como uma extensão do delegate.

Reply:
Done.